### PR TITLE
Jenkins: Fix shell variable check

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Review by furkatgofurov7 — COMMENTED** [review]
lgtm

File: jenkins/scripts/files/run_fetch_logs.sh [hunk 1]
```diff
@@ -71,7 +71,7 @@ fi
 if [[ "${TESTS_FOR}" == "feature_tests" || "${TESTS_FOR}" == "feature_tests_centos" ]]
 then
   target_config=$(sudo find /tmp/ -type f -name "kubeconfig*")
-  if [ "${target_config}" != "" ]
+  if [ ! -z "${target_config}" ]
```
Comment (maelk) [inline jenkins/scripts/files/run_fetch_logs.sh:74]: that should also be equivalent to -n I think (! -z)

**Review by maelk — APPROVED** [review]

**Comment by fmuyassarov** [comment]
For some reason target_cluster logs are not there.

**Comment by kashifest** [comment]
> For some reason target_cluster logs are not there.

Hummmm true, for previous unsuccessful run the logs are there, I guess for successful runs the log fetching is happening after the target cluster is deleted.

**Comment by fmuyassarov** [comment]
@kashifest would you mind to rebase your commit please?

**Pull request merged**